src/Configuration.php

```diff
@@ -214,4 +217,9 @@ public function getAnnotationParser(): AnnotationParser
     {
         return $this->annotationParser;
     }
+
+    public function getLockFilePath(): ?string
+    {
+        return $this->lockFilePath;
+    }
```

Comment:
We could maybe put the logic of deciding the optional lock file path here rather than in TDBMSchemaAnalyzer:

Something like:

```php
    public function getLockFilePath(): string
    {
        return $this->lockFilePath ?: self::getDefaultLockFilePath();
    }
```